src/components/DocumentationTopic/Description/RequirementMetadata.vue

```diff
@@ -11,9 +11,9 @@
 <template functional>
   <!-- because the component is functional we must mannually pass the classes -->
   <p class="requirement-metadata" :class="data.staticClass">
-    <strong>Required.</strong>
+    <strong>{{ parent.$t('required') }}.</strong>
```

Comment:
I missed this spot last time - We should include the punctuation in the localized string. (Although I'm not sure why we even added a period here.)

Reply:
Looking in the source code from `main`, it looks like there are places where we render `"Required."` (with punctuation) and also places where we render `"Required"` (without punctuation).

Reply:
I delete the point